packages/x-adapter-platform/src/types/models/facet.model.ts

```diff
@@ -28,3 +30,29 @@ export interface PlatformFilter {
 export interface PlatformHierarchicalFilter extends PlatformFilter {
   children: PlatformFacet;
 }
+
+/**
+ * Hierarchical Facet model used when combining search response mappers.
+ *
+ * @internal
+ */
+export interface AdapterHierarchicalFacet extends Facet {
```

Comment:
Should be PlatformHierarchicalFacel?

Reply:
This `AdapterHierarchicalFacet` does not really represent the hierarchical facet coming from Platform, it would be the facet resulting from applying the facet schema to a Platform response facet.